proto/proto/geyser.proto

```diff
@@ -157,6 +157,25 @@ message GetHeartbeatIntervalResponse {
   uint64 heartbeat_interval_ms = 1;
 }
 
+message SlotEntryUpdate {
```

Comment:
would be good to link to something like https://github.com/jito-foundation/jito-solana/blob/5396abaad1df66ebcab1e93473ce1b7c9f4c9f6c/entry/src/entry.rs#L124

Reply:
based off of https://github.com/solana-labs/solana/blob/v2.0/geyser-plugin-interface/src/geyser_plugin_interface.rs#L210, added comments